### PR TITLE
all: fix some typos

### PR DESCRIPTION
**Pull request by jtraglia**
Noticed a couple of typos & decided to just fix everything in one PR. Only changed things that I thought were obvious typos. There are a few things I'm a little unsure about though, will add review comments for those.

### CODE REVIEW

File: les/downloader/downloader_test.go [hunk 1]
```diff
@@ -653,8 +653,8 @@ func testForkedSync(t *testing.T, protocol uint, mode SyncMode) {
 	assertOwnForkedChain(t, tester, testChainBase.len(), []int{chainA.len(), chainB.len()})
 }
 
-// Tests that synchronising against a much shorter but much heavyer fork works
-// corrently and is not dropped.
+// Tests that synchronising against a much shorter but much heavier fork works
+// currently and is not dropped.
```
Comment (jtraglia) [inline les/downloader/downloader_test.go:657]: Is this supposed to be `concurrently` or `currently`? Not sure which makes more sense.

File: signer/fourbyte/4byte.json [hunk 2]
```diff
@@ -31711,7 +31711,7 @@
 "1e5296ef": "getMySellOrdersBlackMarketComplete()",
 "1e52aa52": "startBetRed()",
 "1e52ca35": "ROLE_SUPER()",
-"1e52f7b5": "accoutToPayLevel(address)",
+"1e52f7b5": "accountToPayLevel(address)",
```
Comment (jtraglia) [inline signer/fourbyte/4byte.json:31714]: There are four typos in this file. These seem to be external symbols and I'm not sure I should change it.

Should I revert changes to this file?

File: build/ci.go [hunk 3]
```diff
@@ -608,7 +608,7 @@ func doDocker(cmdline []string) {
 			}
 			if mismatch {
 				// Build numbers mismatching, retry in a short time to
-				// avoid concurrent failes in both publisher images. If
+				// avoid concurrent files in both publisher images. If
```
Comment (jtraglia) [inline build/ci.go:611]: Is this supposed to be `fails` or `files`? Just want to be sure.

**Comment by holiman** [comment]
Wow nice job!Yes please revert changes to 4byte.json. Those are abi identifiers.

File: build/ci.go (hunk 3 above)
Comment (MariusVanDerWijden) [reply build/ci.go:611]: I think fails is correct here

File: signer/fourbyte/4byte.json (hunk 2 above)
Comment (jtraglia) [reply signer/fourbyte/4byte.json]: Fixed 👍

**Review by rjl493456442 — APPROVED** [review]
Thanks for your solid work, LGTM!

**Comment by karalabe** [comment]
Gonna generate a nightmare rebase in other PRs. YOLO I guess?

File: accounts/abi/reflect_test.go [hunk 4]
```diff
@@ -32,7 +32,7 @@ type reflectTest struct {
 
 var reflectTests = []reflectTest{
 	{
-		name: "OneToOneCorrespondance",
```
Comment (Frankramos181422) [inline accounts/abi/reflect_test.go:35]: +

**Pull request merged**